test/testUcumLhcUtils.spec.js

```diff
@@ -203,7 +202,7 @@ it("should return a message for invalid unit strings", function() {
     var resp4 = utils.convertUnitTo('mol', 1, '78.4(mmol/L)/s');
     assert.equal(resp4.status, 'failed', resp4.status);
     assert.equal(resp4.msg[0], '4(mmol/L) is not a valid UCUM code.  We assumed you meant 4.(mmol/L).', resp4.msg[0]);
-    assert.equal(resp4.msg[1], 'Sorry.  mol cannot be converted to (78.(4.(mmol/L)))/s.', resp4.msg[1]);
+    assert.equal(resp4.msg[1], 'A mol <-> mol conversion cannot be executed for two mole-based units mol and (78.(4.(mmol/L)))/s.  No conversion was attempted.', resp4.msg[1]);
```

Comment:
I think there should be a comma after "units", e.g., "for two mole-based units, mol and ...".